html/objects/_flag.scss

```diff
@@ -7,45 +7,28 @@
 ////
 
 .sprk-o-Flag {
-  display: table;
-  width: 100%;
-}
-
-.sprk-o-Flag__body,
-.sprk-o-Flag__figure {
-  display: table-cell;
-  vertical-align: top;
+  display: flex;
+  align-items: flex-start;
```

Comment:
Why is the flex-start needed here?

Reply:
you're right, it's not necessary, it's default is flex stretch if not provided and that would be fine for this.